docusaurus/docs/Flutter/guides/adding_localization.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 id: adding_localization
 sidebar_position: 2
-title: Adding Localization
+title: Adding Localization (i18n)
```

Comment:
PR looks good, only question is if we want to add this as internationalization vs l10n